app/src/main/java/org/fossasia/openevent/general/order/OrdersUnderUserViewModel.kt

```diff
@@ -50,11 +48,11 @@ class OrdersUnderUserViewModel(
             }.subscribe({
                 order = it
                 mutableAttendeesNumber.value = it.map { it.attendees?.size } as ArrayList<Int>
-                val query = buildQuery(it)
 
-                if (idList.size != 0)
-                    eventsUnderUser(query)
-                else {
+                val eventIds = it.filter { order -> order.event != null }.map { order -> order.event!!.id }
```

Comment:
Since I'm filtering out `Order`s which have a `null` `Event`, `order.event` will always be non-null in the `map` function and hence the use of `!!` is completely safe.